src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -891,11 +929,11 @@ public Money GetMatureBalance(List<StakeTx> stakeTxes)
         /// and it also has to be mature and meet requirement for minimal value.
         /// </para>
         /// </summary>
-        /// <param name="stakeTxes">List of coins that are candidates for being used for staking.</param>
+        /// <param name="stakeTxes">List of UTXOs that are candidates for being used for staking.</param>
```

Comment:
ok, let's say "UTXO descriptions" so that we are consistent with the comments above